app/src/service/ahgoraService.js

```diff
@@ -1,129 +1,123 @@
+          localStorage.setItem('credential', JSON.stringify(user))
 
-          localStorage.setItem("userDetails", JSON.stringify(userDetails));
-          localStorage.setItem("credential", JSON.stringify(user));
-
-          let comand = `curl https://www.ahgora.com.br/batidaonline/defaultComputer?c=${user.company}`;
+          const comand = `curl https://www.ahgora.com.br/batidaonline/defaultComputer?c=${user.company}`
           window.Neutralino.os.execCommand(comand).then((result) => {
-            localStorage.setItem("identity", result.stdOut);
+            localStorage.setItem('identity', result.stdOut)
           })
 
-
-          resolve(userDetails);
+          resolve(userDetails)
         } else {
-          reject(userDetails);
+          reject(userDetails)
         }
-      });
-    });
+      })
+    })
   },
   getProfileImg: function () {
-    let jwt = JSON.parse(localStorage.getItem("userDetails")).jwt;
-    let profileImg = JSON.parse(localStorage.getItem("userDetails")).employee_id;
-    const header = `cookie: qwert-external=${jwt}`;
-      
-    let comand = `curl -H "${header}" https://www.ahgora.com.br/externo/get_image/${profileImg} --output profile.jpg`;
-    console.log(comand)
-    window.Neutralino.os.execCommand(comand);
-
+    const jwt = JSON.parse(localStorage.getItem('userDetails')).jwt
+    const profileImg = JSON.parse(
+      localStorage.getItem('userDetails')
+    ).employee_id
+    const header = `cookie: qwert-external=${jwt}`
 
+    const comand = `curl -H "${header}" https://www.ahgora.com.br/externo/get_image/${profileImg} --output profile.jpg`
+    console.log(comand)
+    window.Neutralino.os.execCommand(comand)
   },
   espelhoPonto: function (year, month) {
-    let today = new Date();
-    let mirroDate = new Date(year,parseInt(month)-1,1);
+    const today = new Date()
+    const mirroDate = new Date(year, parseInt(month) - 1, 1)
 
     return new Promise((resolve, reject) => {
-      let jwt = JSON.parse(localStorage.getItem("userDetails")).jwt;
+      const jwt = JSON.parse(localStorage.getItem('userDetails')).jwt
 
-      const header = `cookie: qwert-external=${jwt}`;
-      let comand = `curl -H "${header}" -X GET https://www.ahgora.com.br/api-espelho/apuracao/${year}-${month}`;
+      const header = `cookie: qwert-external=${jwt}`
+      const comand = `curl -H "${header}" -X GET https://www.ahgora.com.br/api-espelho/apuracao/${year}-${month}`
 
       window.Neutralino.os.execCommand(comand).then((result) => {
-        let mirror = JSON.parse(result.stdOut);
-        console.log(mirror);
+        const mirror = JSON.parse(result.stdOut)
+        console.log(mirror)
         if (mirror.error) {
-          reject(mirror);
+          reject(mirror)
         } else {
-          localStorage.setItem("mirror", JSON.stringify(mirror));
-          resolve(mirror);
+          localStorage.setItem('mirror', JSON.stringify(mirror))
```

Comment:
Codacy has a fix for the issue: [Replace `'mirror',·JSON.stringify(mirror))` with `"mirror",·JSON.stringify(mirror));`](https://app.codacy.com/gh/ruizivo/ahgora-login/pullRequest?prid=9873926)
```suggestion
          localStorage.setItem("mirror", JSON.stringify(mirror));
```